xmppserver/src/main/webapp/connection-settings-socket-c2s.jsp

```diff
@@ -149,15 +148,15 @@
 
         <p><fmt:message key="ssl.settings.client.plaintext.info"/></p>
 
-        <table cellpadding="3" cellspacing="0" border="0">
-            <tr valign="middle">
-                <td colspan="2"><input type="checkbox" name="plaintext-enabled" id="plaintext-enabled" onclick="applyDisplayable('plaintext')" ${plaintextConfiguration.enabled ? 'checked' : ''}/><label for="plaintext-enabled"><fmt:message key="ssl.settings.client.plaintext.label_enable"/></label></td>
+        <table>
+            <tr>
+                <td colspan="2"><input type="checkbox" name="plaintext-enabled" id="plaintext-enabled" onclick="applyDisplayable('plaintext')" ${plaintextConfiguration.enabled ? 'checked' : ''}/>&nbsp;<label for="plaintext-enabled"><fmt:message key="ssl.settings.client.plaintext.label_enable"/></label></td>
             </tr>
-            <tr valign="middle">
-                <td width="1%" nowrap><label for="plaintext-tcpPort"><fmt:message key="ports.port"/></label></td>
-                <td width="99%"><input type="text" name="plaintext-tcpPort" id="plaintext-tcpPort" value="${plaintextConfiguration.port}"/></td>
+            <tr>
+                <td style="width: 1%; white-space: nowrap"><label for="plaintext-tcpPort"><fmt:message key="ports.port"/></label></td>
+                <td><input type="text" name="plaintext-tcpPort" id="plaintext-tcpPort" value="${plaintextConfiguration.port}"/></td>
```

Comment:
The `valign` or similar looks needed

![image](https://user-images.githubusercontent.com/2117083/203165041-01dda226-c8e6-4b3f-afba-1f89ff0e64b4.png)

Reply:
This appears to be caused by the Bootstrap stylesheet, that adds a `margin-bottom` to labels. I'll override that for labels that are _in_ a table cell.